src/OhadSoft.AzureLetsEncrypt.Renewal/Util/CertificateHelper.cs

```diff
@@ -50,7 +50,8 @@ public static async Task<IReadOnlyList<string>> GetLetsEncryptHostNames(IAzureWe
                 Trace.TraceInformation("Reading ARM certificate query response");
                 var body = await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync().ConfigureAwait(false);
 
-                var letsEncryptCerts = ExtractCertificates(body).Where(s => s.Issuer.Contains(staging ? "Fake LE" : "Let's Encrypt"));
+                var issuerNames = staging ? new[] { "Fake LE" } : new[] { "Let's Encrypt", "R3", "R4", "E1", "E2" };
+                var letsEncryptCerts = ExtractCertificates(body).Where(s => issuerNames.Any(i => s.Issuer.StartsWith(i, StringComparison.InvariantCulture)));
```

Comment:
I think we might as well perform full comparison if we have such a list
So we can change the array above to set (with ordinal comparison), add `“Fake LE Intermediate X1”` and `Let's Encrypt Authority X1` - `Let's Encrypt Authority X4` to it, and do `Contains` instead of `StartsWith`

Reply:
Done. Not sure how to make that testable as it is an internal static helper.  I've moved the issuer names in to fields so they could be set, maybe via the CertificateHelper itself for such a case.